website/app/templates/show.hbs

```diff
@@ -9,9 +9,16 @@
       @extra={{this.extra}}
     />
   {{/if}}
+
   <Doc::Page::Content @breakthrough={{not this.model.hasSidecar}} id={{this.model.showContentId}}>
     <DynamicTemplate @templateString={{this.renderedContent}} @componentId={{this.model.id}} />
+
+    {{#if this.relatedItems}}
+      <h2 class="doc-text-h2">Related</h2>
+      <Doc::Cards::Deck @cols="2" @layout="horizontal" @cards={{this.relatedComponents}} />
+    {{/if}}
```

Comment:
Not sure yet how to only make relatedItems show for "Guidelines" tab. Also need to include link in the Sidecar.

Reply:
We decided just to include it on every tab.